pkg/services/authn/authnimpl/sync/user_sync.go

```diff
@@ -165,7 +165,7 @@ func (s *UserSync) SyncLastSeenHook(ctx context.Context, identity *authn.Identit
 			}
 		}()
 
-		if err := s.userService.UpdateLastSeenAt(context.Background(),
+		if err := s.userService.UpdateLastSeenAt(ctx,
```

Comment:
So because this is running in a gorutine it is not guaranteed to be completed before the request ends and risk getting canceled.

I think to be on the safe side we should either not run it in a separate goroutine or use [context.WithoutCancel](https://pkg.go.dev/context#WithoutCancel)

Reply:
Good point!